talk/throttle_latch/throttling/talker4.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import os
+import rospy
+from std_msgs.msg import String
+
+RATE = float(os.getenv("TOPIC_RATE_4", 10))
+
+def talker4():
```

Comment:
remove multiple talker<no>.py file and use a single file you can specify the topic rate as environment variable http://wiki.ros.org/roslaunch/XML